service/spring.gradle

```diff
@@ -1,28 +1,29 @@
 configurations { compileOnly { extendsFrom annotationProcessor } }
 
-var springBootVersion = '2.7.4'
+var springBootVersion = '3.1.6'
 
 dependencies {
+	implementation "org.springframework.boot:spring-boot-starter-actuator:$springBootVersion"
```

Comment:
The spring dependency plugin is supposed to eliminate the need to specify the version of spring components like spring-boot. If you update the dependency plugin to the current version, can you remove the versions from this dependency?

Reply:
Yep!